### ISSUE TITLE
Balance only updates when I reload the wallet app

### ISSUE THREAD

**Issue by kantai** [blockstack/blockstack-browser#543]
I experience this running both in regtest and on production. I need to go to the "wallet app" after I've sent money for my balance to update.

**Comment by scott-martin** [comment]
I've also experienced issues with this. For example, I transferred some BTC to my wallet yesterday. This morning when I loaded up my wallet for the first time I had to go back and forth to the wallet app about 5 times before the new balance showed up.

**Comment by larrysalibra** [comment]
Thanks @scott-martin @kantai ! I'm going to take a look at this this morning.

**Comment by kantai** [comment]
Another related, but different problem, with respect to wallet balances is that once a user has issued a registration, if the user only had 1 UTXO, that UTXO is used in the name registration, and the change UTXO hasn't been confirmed yet, so the user's balance will display (correctly) 0. This is more of a UX problem than a bug, and I don't have any obvious suggestions for how to indicate to a user that they're waiting on a name or a UTXO to confirm. We can move this to a separate issue if you think that's better.
